static/src/javascripts/projects/common/modules/identity/api.ts

```diff
@@ -13,42 +16,11 @@ type UserNameError = {
 	context: string;
 };
 
-type UserConsents = {
-	id: string;
-	actor: string;
-	version: number;
-	consented: boolean;
-	timestamp: string;
-	privacyPolicyVersion: number;
-};
-
-type UserGroups = {
-	path: string;
-	packageCode: string;
-};
-
 export type IdentityUser = {
```

Comment:
We wanted to be able to [return](okta/refactor-getUserFromApi) an object of type `IdentityUser` both when `SignedInWithCookies` and `SignedInWithOkta` so we've simplified the type by deleting the properties that do not seem to be used anywhere.  We now construct `IdentityUser` with data from IDAPI or Okta's `accessToken` and
`idToken`.

Reply:
Yeah this is looking good, in Okta we now only return fields that the application needs, and if this is all frontend is using then that's great!